WalletWasabi.Fluent/Views/Settings/SettingsPageView.axaml

```diff
@@ -14,7 +14,7 @@
                  Caption="Manage appearance, privacy and other settings"
                  EnableNext="True"
                  EnableCancel="{Binding !IsReadOnly}"
-                 NextContent="Done"
+                 NextContent="Save"
```

Comment:
Adding more unrelated changes won't make it easier to merge, instead it can create more controversy and distractions.